resource/source-python/translations/wcs/chat_strings.ini

```diff
@@ -2,277 +2,352 @@
+    da = "{lightgreen}Du gav alle online spillere {green}{value} level(s){lightgreen}."
 [admin gain levels receiver]
     en = "{lightgreen}You've gained {green}{value} level(s) {lightgreen}from {green}{name}{lightgreen}."
     ru = "{lightgreen}Вы получили {green}{value} уровней {lightgreen}от {green}{name}{lightgreen}."
     fi = "{lightgreen}Sinä siat {green}{value} levelei {lightgreen} {green}{name}{lightgreen}."
+    da = "{lightgreen}Du fik {green}{value} level(s) {lightgreen}af {green}{name}{lightgreen}."
 [admin gain levels sender]
     en = "{green}{name} {lightgreen}have gained {green}{value} level(s){lightgreen} from you."
     ru = "{green}{name} {lightgreen}получил {green}{value} уровней{lightgreen} от Вас."
     fi = "{green}{name} {lightgreen}on saannut {green}{value} levelei{lightgreen} sinulta."
+    da = "{green}{name} {lightgreen}fik {green}{value} level(s){lightgreen} af dig."
 [admin gain levels self]
     en = "{lightgreen}You have given yourself {green}{value} level(s){lightgreen}."
     ru = "{lightgreen}Вы дали себе {green}{value} уровней{lightgreen}."
     fi = "{lightgreen}Sinä olet antanut itsellesi {green}{value} levelei{lightgreen}."
+    da = "{lightgreen}Du gav dig selv {green}{value} level(s){lightgreen}."
 [admin changerace receiver]
     en = "{lightgreen}You've had your race changed to {green}{value}{lightgreen}."
     ru = "{lightgreen}Вы изменили свою расу на {green}{value}{lightgreen}."
     fi = "{lightgreen}Sinun racesin on vaihdettu {green}{value}{lightgreen}."
+    da = "{lightgreen}En administrator skiftede din race til {green}{value}{lightgreen}."
 [admin changerace sender]
     en = "{green}{name} {lightgreen}have changed race to {green}{value}{lightgreen}."
     ru = "{green}{name} {lightgreen}сменил расу на {green}{value}{lightgreen}."
     fi = "{green}{name} {lightgreen}on vaihdettu race {green}{value}{lightgreen}."
+    da = "{green}{name} {lightgreen}skiftede race til {green}{value}{lightgreen}."
 [admin changerace self]
     en = "{lightgreen}You've changed your race to {green}{value}{lightgreen}."
     ru = "{lightgreen}Вы сменили расу на {green}{value}{lightgreen}."
     fi = "{lightgreen}Sinä vaihdoit sinun racesi {green}{value}{lightgreen}."
+    da = "{lightgreen}Du skiftede din race til {green}{value}{lightgreen}."
 [admin gain bank levels all]
     en = "{lightgreen}You have given all online players {green}{value} level(s){lightgreen} to their bank."
-    fi = "{lightgreen}Sinä olet antanut kaikkille paikalla oleville pelaajille {green}{value} levelei{lightgreen} heidän pankkiin."
+    ru = ""
+    fi = ""
+    da = "{lightgreen}Du tilføjede{green}{value} level(s){lightgreen} til alle online spilleres wcsbank konti."
 [admin gain bank levels receiver]
     en = "{lightgreen}You've gained {green}{value} level(s) {lightgreen}to your bank from {green}{name}{lightgreen}."
-    fi = "{lightgreen}Sinä sait {green}{value} levelei {lightgreen}sinun pankkiin {green}{name}:ltä{lightgreen}."
```

Comment:
This need add back

Reply:
Hi srpg!
Thanks for letting me know, I'm a bit of a newbie when it comes to GitHub, I will take the strings you've posted in here and include them in my strings and then I'll update it again, I never meant to overwrite your work (^.^)